.env

```diff
@@ -9,6 +9,7 @@ STACKS_CORE_EVENT_HOST=127.0.0.1
 
 STACKS_BLOCKCHAIN_API_PORT=3999
 STACKS_BLOCKCHAIN_API_HOST=127.0.0.1
+STACKS_BLOCKCHAIN_API_DB=pg
```

Comment:
Do we want to commit this?

Reply:
Well ideally we should not commit `.env` file at all. See the recommendation from the npm package used https://github.com/motdotla/dotenv#should-i-commit-my-env-file

My proposition is to have `example.env` file that could be used as a template and just let it loudly fail if you don't have setup your environment variables.

What do you think ?

Reply:
But yeah it was a mistake it should not be there. I just modified it to fit my need.

Reply:
I actually don't agree with the `dotenv` recommendations in regards to this repo. The `.env` file serves as a single source of truth reference for various env vars and the default values. The alternative is to essentially duplicate the contents of this file as JS code within some more obscure init area, then also duplicate the contents of this file in another readme doc somewhere -- introducing yet another area to maintain, that will inevitably end up with bugs or outdated info. 

The `example.env` approach could work, but then we'd have to add documentation for `rename example.env to .env`, which pretty much everyone would be required to do, and kinda defeats the purpose.

Reply:
The `.env` file define environment-specific values and there is passwords (postgres and btc rpc). I think that adding to the documentation an instruction about properly setting your environment variables is a good thing.

Reply:
I have removed it.